include/klee/Statistic.h

```diff
@@ -25,22 +23,21 @@ namespace klee {
   /// not the actual values. Values are managed by the global
   /// StatisticManager to enable transparent support for instruction
   /// level and call path level statistics.
-  class Statistic {
+  class Statistic final {
     friend class StatisticManager;
     friend class StatisticRecord;
 
   private:
-    unsigned id;
+    std::uint32_t id = 0;
```

Comment:
Can you remove the initialization?  I think it's misleading.

Reply:
@ccadar
I've added it because modern IDEs complain about an uninitialised member (it's not initialised via ctor). Shall I still remove it or add a comment?